.github/scripts/compile.sh

```diff
@@ -106,33 +106,49 @@ do_build () {
   local suffix=${1}
   local runtime_check=${2}
 
+  if [ "$runtime_check" == "OFF" ]; then
+     # Turn on PGO only when we are not doing asan or tsan
+     BUILD_OPTS="-DENABLE_PROFILE_GUIDED_OPTIMIZATION=ON"
+  else
+     BUILD_OPTS="-DBUILD_TESTING=OFF"
+  fi
+
   cmake -S "${PROTON_DIR}" -B "${PROTON_BUILD_DIR}${suffix}" \
     -DCMAKE_BUILD_TYPE=RelWithDebInfo \
     -DRUNTIME_CHECK="${runtime_check}" \
     -DENABLE_LINKTIME_OPTIMIZATION=ON \
-    -DCMAKE_POLICY_DEFAULT_CMP0069=NEW -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON \
+    -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON \
     -DBUILD_TLS=ON -DSSL_IMPL=openssl -DBUILD_STATIC_LIBS=ON -DBUILD_BINDINGS=python \
     -DBUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF \
     -DCMAKE_INSTALL_PREFIX=/usr
   cmake --build "${PROTON_BUILD_DIR}${suffix}" --verbose
 
   DESTDIR="$PROTON_INSTALL_DIR${suffix}" cmake --install "$PROTON_BUILD_DIR${suffix}"
 
+   if [ "$runtime_check" == "OFF" ]; then
+     # Install Proton Python
+     python3 -m pip install --prefix="$PROTON_INSTALL_DIR/usr" "$(find "$PROTON_BUILD_DIR/python/" -name 'python-qpid-proton-*.tar.gz')"
+
+     # This will install the proton python libraries in sys.path so the tests using
+     # proton can be run successfully.
+     python3 -m pip install "$(find "$PROTON_BUILD_DIR/python/" -name 'python-qpid-proton-*.tar.gz')"
+   fi
```

Comment:
I'd place this unconditionally before the

```
# Do a regular build without asan or tsan.
do_build "" OFF
```

(at line 133/151)

Reply:
The `# Install Proton Python` can remain where it is, after the builds. That installs proton for the output image.